src/query-helpers.js

```diff
@@ -29,6 +29,13 @@ function getElementError(message, container) {
   return new Error([message, debugDOM(container)].filter(Boolean).join('\n\n'))
 }
 
+function getGetByElementError(message, container) {
+  return getElementError(
+    `${message}\n\n(If this is intentional, then use the \`*AllBy*\` variant of the query (like \`getAllByText\` or \`findAllByText\`)).`,
```

Comment:
This error message is nice and clear and makes the breaking change pretty safe

Reply:
Might be able to enhance it even more with another parameter like `query`, which contains the suffix, `AllByText`. Then this could be updated to:

```
`${message}\n\n(If this is intentional, then use the \`get${query}\` or \`find${query}\` variant of the query).`
```